core/code/hooks.js

```diff
@@ -50,8 +50,8 @@
 //              displayed. The data hash contains both the unprocessed
 //              raw ajax response as well as the processed chat data
 //              that is going to be used for display.
-//     WARNING: this hook was disabled in earlier versions
-//              sideeffects possible but unknown. Use with care!
+//              This hook was implemented in earlier versions
+//              but commented out and never used before.
```

Comment:
You better remove this comment completely. It is not helpful at all.

In general, providing new API (such as hook) is a big responsibility: you must to make sure that you are providing some functionality in best (and complete) way. Because you will not be able to fix this after it is already used in dozens of plugins.
So are you sure? If you are - good. If not - you should not uncomment that part prematurely.

Reply:
We need that hook learn about the MSGs in ALERScomm.
It has been set up in the same way as for the ALLcomm.
This comment was already changed on request of @modos189.
Please agree on what you prefer and edit accordingly.

Reply:
> We need that hook learn about the MSGs in ALERScomm

No problem - implement whatever you need. Just take enough time to make sure that it is implemented in best way, as already said.
And after you will be sure - those stupid comment will not be needed.